dataflow/tests/reachingdefinitions/Test.java

```diff
@@ -0,0 +1,13 @@
+public class Test {
+    public int test() {
+        int a = 1, b = 2, c = 3;
```

Comment:
suggestion: maybe test compound assignment as well?

Reply:
Yeah, but I think it could be better to add such test cases after my PR remove stringConcatenateAssignmment merge..

Reply:
Sure.